packages/consent/consent-tools/src/types/settings.ts

```diff
@@ -27,6 +27,36 @@ export interface CreateWrapperSettings {
    **/
   getCategories: () => Categories | Promise<Categories>
 
+  /**
+   * Programatically send a "Segment Consent Preference" event to Segment when consent preferences change.
+   * An event will be sent to Segment whenever this function is called.
```

Comment:
I would specify that this is a "Function to register a listener for consent changes to programatically send ..."